ansys/mapdl/core/mapdl.py

```diff
@@ -185,7 +185,28 @@ def post_processing(self):
 
     @property
     def chain_commands(self):
-        """Chain several mapdl commands."""
+        """Chain several mapdl commands.
+
+        Commands can be separated with ``"$"`` in MAPDL rather than
+        with a line break, so you could send multiple commands to
+        MAPDL with:
+
+        ``mapdl.run("/PREP7$K,1,1,2,3")``
+
+        This method is merely a convenience context manager to allow
+        for easy chaining of PyMAPDL commands to speed up sending
+        commands to MAPDL.
+
+        .. note::
+            Commands must be limited to 640 characters.
```

Comment:
Per command or total?

Reply:
This is actually an implementation detail, so I've removed it.  The context manager handles this for the user by splitting lines that exceed 640 characters.